dialog/src/main/java/roboy/util/QAJsonParser.java

```diff
@@ -1,73 +1,129 @@
 package roboy.util;
 
-import com.google.gson.Gson;
-import com.google.gson.JsonIOException;
-import com.google.gson.JsonParseException;
-import com.google.gson.JsonSyntaxException;
+import com.google.common.base.Charsets;
```

Comment:
remove `QAFileParser.java` then

Reply:
<img width="336" alt="screen shot 2018-09-17 at 20 07 07" src="https://user-images.githubusercontent.com/7304718/45641270-4ab4a600-bab5-11e8-87ac-63a9205bbafb.png">
I removed it